From a6be02ad4b564d2079681589f4862c32419c2c9f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Joachim=20Mairb=C3=B6ck?= <j.mairboeck@gmail.com>
Date: Sat, 30 Aug 2025 19:55:16 +0200
Subject: fix test build

Haiku doesn't support WCOREDUMP yet, but WIFCORED should be equivalent.

diff --git a/tests/t_dumpload.c b/tests/t_dumpload.c
index 6cfc74c..f5176dc 100644
--- a/tests/t_dumpload.c
+++ b/tests/t_dumpload.c
@@ -85,7 +85,7 @@ createdb (char const *name)
   else if (WIFSIGNALED (rc))
     {
       fprintf (stderr, "%s: terminated on signal %d", cmd, WTERMSIG (rc));
-      if (WCOREDUMP (rc))
+      if (WIFCORED (rc))
 	fprintf (stderr, " (core dumped)");
       fputc ('\n', stderr);
       exit (1);
-- 
2.50.1

